### PR TITLE
Bug 1767780: Use labelPlural for k8s resource inventory items

### PR DESCRIPTION
**Pull request by rawagner**

### CODE REVIEW

File: frontend/packages/console-shared/src/components/dashboard/inventory-card/InventoryItem.tsx [hunk 1]
```diff
@@ -59,10 +59,10 @@ const getStatusGroupIcons = (flags: FlagsObject) => {
 };
 
 const InventoryItem: React.FC<InventoryItemProps> = React.memo(
-  ({ isLoading, title, count, children, error = false, TitleComponent, ExpandedComponent }) => {
+  ({ isLoading, title, titlePlural, count, children, error = false, TitleComponent, ExpandedComponent }) => {
     const [expanded, setExpanded] = React.useState(false);
     const onClick = React.useCallback(() => setExpanded(!expanded), [expanded]);
-    const pluralizedTitle = count !== 1 ? `${title}s` : title;
+    const pluralizedTitle = count === 1 ? title : (titlePlural || `${title}s`);
```
Comment (spadgett) [inline frontend/packages/console-shared/src/components/dashboard/inventory-card/InventoryItem.tsx]: We should use the existing `pluralize` utility:

https://github.com/spadgett/console/blob/cc95eed613998a265a48e6bb823c04940edf63da/frontend/public/components/utils/details-page.tsx#L18-L19

Comment (rawagner) [reply frontend/packages/console-shared/src/components/dashboard/inventory-card/InventoryItem.tsx]: PR updated - using `pluralize` now

**Comment by spadgett** [comment]
/lgtm cancel

lint error

**Pull request merged**